### PR TITLE
[feat] atomic design folder structure for components

### PR DESCRIPTION
**Pull request by raj-kumar-litmus**
1. Components have been organised into atomic structure.

- atoms - the root components.
- molecules - two or more atomic components combine to form molecular components.
- organism - atoms and or molecules combine to form organisms.

2. sampleJSON folder has been deleted.
3. views folder contains components corresponding to each route.
4. Helpers directory to subsume constants, utils, config

### CODE REVIEW

File: src/components/atoms/CustomIcon.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { FC } from "react";
 import { CustomIconProps } from "../../@types/components/commonTypes";
-
+//todo for Sarath. remove this component.
```
Comment (sarath-sasi-panickaveetil) [inline src/components/atoms/CustomIcon.tsx]: Removed customIcon component in PR #68 . just fyi

File: src/components/atoms/InputText.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    >
+      {props.label && (
+        <label className="relative text-xs font-medium pb-1 text-gray-400">
+          {props.label}
+        </label>
+      )}
+      {props.icon && (
+        <CustomImage
+          src={props.icon}
+          className={`absolute top-8 left-2 ${props.imageclassname}`}
```
Comment (sarath-sasi-panickaveetil) [inline src/components/atoms/InputText.tsx:19]: There will be 'undefined' text if 'imageclassname' is undefined

File: src/components/atoms/InputText.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        />
+      )}
+      <InputText
+        {...props}
+        className={`${props.icon ? "p-2 pl-8" : ""} ${props.className}`}
```
Comment (sarath-sasi-panickaveetil) [inline src/components/atoms/InputText.tsx:24]: same as previous comments. handle the case when 'className' become 'undefined'

File: src/components/atoms/InputText.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { InputText } from "primereact/inputtext";
+import React from "react";
+import CustomImage from "./customimage";
+import { CustomInputTextProps } from "../../@types/components/commonTypes";
+
+const CustomInputText: React.FC<CustomInputTextProps> = (props) => {
+  return (
+    <div
+      className={`flex flex-col w-full relative ${props.containerclassname} self-end `}
```
Comment (AdithyaShaj) [inline src/components/atoms/InputText.tsx:9]: use ${props.containerclassname || ""}

File: src/components/molecules/CustomCalendar.tsx [hunk 5]
```diff
@@ -100,9 +103,11 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
       >
         {props.title}
       </div>
-      <img className={`absolute ${props.imageclassname}`} src={props.imgsrc} />
-      <Calendar    
-       
+      <CustomImage
+        src={props.imgsrc}
+        className={`absolute ${props.imageclassname}`}
```
Comment (AdithyaShaj) [inline src/components/molecules/CustomCalendar.tsx:108]: Same comment as above

**Pull request closed without merging**